cypress/integration/calculator.spec.js

```diff
@@ -0,0 +1,52 @@
+describe('계산기 테스트', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:5500/');
+  });
+
+  it('2개의 숫자에 대해 덧셈이 가능하다.', () => {
+    cy.get('.digit').contains('2').click();
```

Comment:
```suggestion
    const handleClickNumber = (number) => cy.get('.digit').contains(number).click();
    
    handleClickNumber(1);
    handleClickNumber(2);
```
이렇게 함수로 만들어서 사용하면 어떨까요? 그럼 더 직관적일 것 같아요!
더 나아가서

```suggestion
    const handleClickNumber = (...numbers) => numbers.forEach(number => cy.get('.digit').contains(number).click());
    
    handleClickNumber(1, 2); // 12
```
이렇게 하면 여러 개의 숫자를 누를 수 있도록 표현할 수 있답니다.!

Reply:
이런식으로 사용하는것은 숫자를 클릭할때는 handleClickNumber함수에서  관리하기 위한건가요 ??

Reply:
넵 맞습니다! 그리고 코드가 어떤 역할을 하는지 명확하게 의도적으로 표현할 수 있답니다 ㅎㅎ